docs/cli.md

```diff
@@ -74,8 +74,8 @@ Discord (chat): https://discord.gg/rzdnErX
 
    ```
    
-   
-#### Wasp has a set of commands for working with the db. They all start with `db` and mostly call prisma commands in the background.
+### Database 
```

Comment:
I thikn you want to put `####` here, in front of `Database`, and remove any hashes in front of the paragraph below `Wasp has a set of commands for working ...`.

Reply:
Yeah my mistake!